scripts/check_langfile_duplicate_identifier.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+. scripts/Import/Functions.sh
+
+exit_status=0
+
+if [ -z "${GHRUN}" ]
+then
+  langfiles=$(git diff --cached --name-only --diff-filter=ACM -- '*.lang')
+else
+  langfiles=$(get_changed_lang_files)
+fi
+
+for file in $langfiles
+do
+  # Find the duplicate identifiers
+  duplicate_identifiers=$(sed -n '/<!-- language file start -->/,$p' $file | awk -F '#:#' '{print tolower($2)}' | sort -f | uniq -d)
```

Comment:
Sorry, I missed this in the previous review: Please quote the variable `$file` to prevent argument expansion:

```suggestion
  duplicate_identifiers=$(sed -n '/<!-- language file start -->/,$p' "$file" | awk -F '#:#' '{print tolower($2)}' | sort -f | uniq -d)
```

If this is changed, I happily approve this PR :)

Reply:
@lscharmer Thank you!